core/src/components/progress-bar/test/shape/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="UTF-8" />
+    <title>Progress Bar - Shape</title>
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no"
+    />
+    <link href="../../../../../css/ionic.bundle.css" rel="stylesheet" />
+    <link href="../../../../../scripts/testing/styles.css" rel="stylesheet" />
+    <script src="../../../../../scripts/testing/scripts.js"></script>
+    <script nomodule src="../../../../../dist/ionic/ionic.js"></script>
+    <script type="module" src="../../../../../dist/ionic/ionic.esm.js"></script>
+
+    <style>
+      ion-content::part(scroll) {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+      }
+
+      ion-progress-bar {
+        margin: 10px 0;
```

Comment:
After reading your comment, I realized that the other pages uses the margin. So it does make sense to keep. Just making a suggestion to be consistent.

```suggestion
      ion-content ion-progress-bar {
        margin: 10px 0;
```